src/dune_pkg/dev_tool.mli

```diff
@@ -0,0 +1,5 @@
+type cmd = Fmt
```

Comment:
Rather than calling this module "dev_tool", call it ocamlformat, since it's really specific to ocamlformat

Reply:
This is because the other dev-tool like `odoc`, could be added in `dev_tool`.

Reply:
When they will be needed, they can be added and the module be re-introduced. So far, I see nothing generic in this module that warrants such a generic name.